birdhouse/scripts/get-services-json.include.sh

```diff
@@ -12,7 +12,16 @@ fi
 
 for adir in ${ALL_CONF_DIRS}; do
   [ -f "${adir}/service-config.json" ] || continue
-  SERVICES="${SERVICES}$([ -n "${SERVICES}" ] && echo ',') $(cat "${adir}/service-config.json")"
+  # read and strip leading/trailing whitespaces
+  SERVICE_CONF="$(cat "${adir}/service-config.json" | sed -z 's/^\s*//;s/\s*$//')"
+  # merge whether config is an array or a single service
+  if [ "$(echo "${SERVICE_CONF}" | head -n 1)" = "[" ]; then
```

Comment:
If you just left the stac/service-config.json.template file as is, all of this wouldn't be necessary.

I don't understand why this is needed

Reply:
Either this, or we patch everywhere that tries to load the JSON. 
I add errors when trying to extract the `$schema` in the tests.

Reply:
I purposely went without the `jq` approach to avoid introducing this extra dependency since the merge is relatively simple in this case.

One issue with the pseudo-JSON is that it is not obvious (for others) that the incorrectly formed content is intentional to work only once merged. Especially when working with IDEs, or investigating the generated JSON on the server, we get flagged about invalid JSON, which doesn't help maintain them. Since there is a way to make them valid JSON definitions and still work for the aggregation, I would favor using a valid list.